README.md

```diff
@@ -12,7 +12,7 @@ The site is built and deployed automatically from the repo (see .github/workflow
 
 To preview/experiment locally:
 
-1) [Install Zola][zola-get].
```

Comment:
maybe it's better to link ci.yml here for future-proofing?

Reply:
I think it's worth having the version number in the README so people don't have to read through the CI config, but I think it'd be good to link it here too :)